packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -153,12 +164,37 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
     }
   };
 
+  const createContextValue = useMemo(
```

Comment:
Couldn't this be simplified somewhat by using `useCallback()` rather than `useMemo()` since the return value is basically a function?